docs/demo/index.html

```diff
@@ -11,12 +11,18 @@
 <body>
    <h1>Demo of <code>silvermine-videojs-airplay</code></h1>
 
-   <video id="video_1" class="video-js vjs-default-skin" controls preload="auto" data-setup='{ "fluid": "true" }'>
+   <video id="video_1" class="video-js vjs-default-skin" controls preload="auto">
       <source src="http://www.caminandes.com/download/03_caminandes_llamigos_1080p.mp4" type="video/mp4">
    </video>
 
    <script>
-      videojs("video_1", {}, function() {
+      var options;
+
+      options = {
+         fluid: true,
+      };
+
+      videojs("video_1", options, function() {
```

Comment:
Nit: can we use single quotes here to match our coding standards? Sorry - there's no linting in these HTML files to catch it.